docs/content/latest/quick-start/install/linux.md

```diff
@@ -50,7 +50,7 @@ showAsideToc: true
 
 ## Prerequisites
 
-1. One of the following operating systems
+1. One of the following operating systems:
 
     * <i class="icon-centos"></i> CentOS 7
```

Comment:
I wonder if we should also mention Oracle Linux here, and possibly others. Someone was asking for that to be added on the Replicated install page recently (see #8492)

Reply:
fixed